gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtils.java

```diff
@@ -397,15 +390,15 @@ static List<String> getDistinctUniqueRequesters(String serializedRequesters) {
     }
   }
 
-  public static void submitInitializationEventsAndSetStatus(Dag<JobExecutionPlan> dag, Optional<EventSubmitter> eventSubmitter) {
-    if (eventSubmitter.isPresent()) {
+  public static void submitAndSet(Dag<JobExecutionPlan> dag, Optional<EventSubmitter> eventSubmitter) {
```

Comment:
what did you think of my prior suggestion to lift the conditional processing out of this method?  instead it would always take action and we'd leave it to the caller to evaluate the `Optional<EventSubmitter>` and decide whether or not to call this method